docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -0,0 +1,43 @@
+get:
+  description:
+    List all running tasks.
+  is: [ secured ]
+  queryParameters:
+    status:
+      description: Filter the list of tasks by status
+      enum: [ TASK_STAGING, TASK_STARTING, TASK_RUNNING, TASK_FINISHED, TASK_FAILED, TASK_KILLED, TASK_LOST, TASK_ERROR ]
+      repeat: true
+  responses:
+    200:
+      description: The list of running tasks.
+      body:
+        application/json:
+          example: !include examples/tasks.json
+        text/plain:
+          example: |
+            frontend-foosball	10019	srv4.example.com:31292
+            tools_docker_registry	5000	srv2.example.com:31721
+
+
+/delete:
+  post:
+    description:
+      Kill a list of running tasks.
+    queryParameters:
+      scale:
+        type: boolean
+        description: If `scale=true` is specified, then the related application is scaled down by the number of killed tasks.
+        default: false
+    body:
+      application/json:
+        example: |
+          { "ids": [ "task1", "task2" ] }
+    responses:
+      200:
+        description: No content is send with this reponse.
```

Comment:
Should we document this: if `scale == true`, a `DeploymentPlan` will be created for each app corresponding to one or more of the given tasks. These are not returned, so one cannot track the deployments in order to see when the killing of multiple tasks with scaling eventually finished.

btw: I am not convinced that this functionality makes sense and it used (killing tasks of multiple apps at once while scaling their instance count to whatever remains). Maybe we should deprecate that and see if anyone complains.